docs/resources/log_stream.md

```diff
@@ -60,7 +60,7 @@ resource "auth0_log_stream" "example_aws" {
 
 - `name` (String) Name of the log stream.
 - `sink` (Block List, Min: 1, Max: 1) The sink configuration for the log stream. (see [below for nested schema](#nestedblock--sink))
-- `type` (String) Type of the log stream, which indicates the sink provider.
+- `type` (String) Type of the log stream, which indicates the sink provider. Possible values are: `eventbridge`, `eventgrid`, `http`, `datadog`, `splunk`, `sumo`, `mixpanel`.
```

Comment:
i pulled these from the provider's validation response:  Error: expected type to be one of [eventbridge eventgrid http datadog splunk sumo mixpanel], got <xyz>

Reply:
Hey @ginman86 👋🏻 

Thanks for the contribution! This is definitely a good addition! 

Would you mind updating the description for this here https://github.com/auth0/terraform-provider-auth0/blob/80cce7224e75d3476b1ba7f14b25ac4bbf1b777a/internal/provider/resource_auth0_log_stream.go#L48 and then run `make docs` ? As the content within the docs folder is completely generated and shouldn't be modified manually. Additionally, we also included support for `segment` log stream. 

To give better guiding on how to contribute we updated as well our CONTRIBUTING.md docs: https://github.com/auth0/terraform-provider-auth0/blob/patch/contributing/CONTRIBUTING.md#documentation (pending to be merged in).